src/pages/product/tech-insights.js

```diff
@@ -0,0 +1,293 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import has from 'lodash/has';
+import { SEO, SitewideFooter, SitewideHeader, Button, Headline } from 'components';
+import { AlternatingFeatureBlock, AlternatingFeatureWrapper } from 'components/landing';
+import Title from '../../components/Title';
+
+import TechInsightsScreenshotImg from '../../../content/assets/product-pages/techinsights-hero.svg';
+import CreateCheckImg from '../../../content/assets/product-pages/techinsights-create-check.svg';
+import IntegrationsImg from '../../../content/assets/product-pages/techinsights-integrations.png';
+import GitHubBotImg from '../../../content/assets/product-pages/techinsights-github-bot.svg';
+import Logi4jImg from '../../../content/assets/product-pages/techinsights-logi4j.svg';
+
+import { LOGOS } from '../../components/landing/CustomerLogoCloud';
+import { Tab, TabList, TabPanel, Tabs } from 'react-tabs';
+import classNames from 'classnames';
+
+const SEO_TITLE = 'Tech Insights: scorecards for Backstage [coming soon]';
+const LEAD = `Roadie’s Backstage-based Tech Insights lets you measure and improve software quality`;
+
+const PRODUCT = {
+  features: [
+    {
+      title: 'Raise the bar',
```

Comment:
This is too ambiguous. You need headers to have very concrete value props and how your product will provide it. This title and subtitle could be used to offer coaching or consulting for instance, or for many other kinds of products